docs/source/procedures/datadog/index.rst

```diff
@@ -40,6 +40,19 @@ Upload the Dashbaord
 Download the dashboard file :download:`dashboard.json <dashboard.json>`.
 Create a new dashbaord in Datadog and import the json file you downloaded. 
 
+Using the Dashbaord
+===================
+We created a datadog dashboard that resembles the Grafana dashboards available to Scylla and Scylla-Cloud users.
+
+It contains some specific filtering and perspectives that are not always available in Datadog.
```

Comment:
```suggestion
It contains specific filtering and perspectives that are only sometimes available in Datadog.
```

Reply:
Putting the wording aside, I do not undertsatd what this means.
What features are "not always available in Datadog"? why are they not available?

Reply:
The option to have a shar/dc/cluster view is not so easy todo in Datadog

Reply:
- Its not clear from the wording
- why is it important to the reader?

I sugget to remove this

Reply:
The main reason that I'm sending the pull request is because a user created such a combination that they had no data.

Reply:
Again, "It contains some specific filtering and perspectives that are not always available in Datadog" is not clear.
It look like the filtering and perspectives are not available in Datadog.
AFAIU, the *metrics* are not visable.

This is well describe in the note below.

Reply:
I'll remove it, the different perspective is something that is not typically exists in DD, I jumped through a few hoops to make it working and without prometheus backing me up it wouldn't have worked.